free-1.md

```diff
@@ -57,12 +164,28 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The implementer would be able to leverage existing solutions for much of the
+systems aspect. As an example, a prototype of this language could be a compiler
+or transpiler that outputs CSound code, thereby using the existing CSound
+infrastructure for actually generating audio. This leaves plenty of time for
+language decisions to be made.
```

Comment:
I agree. I think it would be interesting to see whether a graphical syntax has enough benefit to the user over a text-based syntax that would merit spending extra time. These decisions Jeb has to make probably indicates the start of a good project.